pkg/power/accelerator/source/gpu_nvml.go

```diff
@@ -57,7 +57,7 @@ func (n *GPUNvml) Init() (err error) {
 		err = fmt.Errorf("failed to get nvml device count: %v", nvml.ErrorString(ret))
 		return err
 	}
-	klog.V(1).Infof("found %d gpu devices\n", count)
+	fmt.Printf("found %d gpu devices\n", count)
 	devices = make([]interface{}, count)
```

Comment:
we agreed to switch from fmt to klog
why need this change back?

Reply:
this is called in `init` function, klog is not initialized yet so klog Infof doesn't print anything at this time.

Reply:
ok, thanks for point that out